Documentation/spec/docfx_flavored_markdown.md

```diff
@@ -152,3 +152,106 @@ The above content will be transformed to the following html:
   <p>WARNING content</p>
 </div>
 ```
+
+Difference between DFM and GFM
+------------------
+
+### List
+
+DFM list item always uses the first character in the first line to decide whether it is in the list item, but GFM does not.
+For example:
+Case 1:
+```md
+* a
+  * b
+    * c
+
+  text
+```
+
+`a` starts at column 3, `b` starts at column 5, `c` starts at column 7.  
```

Comment:
202 and 242 require <br>

---
In reply to: [88595260](https://github.com/dotnet/docfx/pull/905#discussion_r88595260) [](ancestors = 88595260)